example/basic_bot/main.go

```diff
@@ -140,7 +140,7 @@ func main() {
 	)
 
 	lang := os.Getenv("NOSTALE_SERVER_LANG")
-	servers, err := userInteractor.Login(user, lang, getCountryID(lang))
+	servers, err := userInteractor.Login(user, os.Getenv("NOSTALE_JSON_ACCOUNT"), os.Getenv("NOSTALE_JSON_IDENTITY"), getCountryID(lang))
```

Comment:
Should get the `User` information here and pass it down to the interactor instead of passing just the json path :) more info below.